package.json

```diff
@@ -17,21 +17,22 @@
   "main": "./lib/index.js",
   "types": "./typings/index.d.ts",
   "files": [
-    "lib",
-    "typings"
+    "dist"
   ],
   "scripts": {
-    "test": "nyc --extension --reporter=lcov --reporter=text-summary vows -- --spec test/*-test.js"
-  },
-  "directories": {
-    "lib": "./lib"
+    "test": "nyc --extension --reporter=lcov --reporter=text-summary vows -- --spec test/*-test.js",
+    "build" : "tsc",
+    "prepare" : "tsc"
   },
   "devDependencies": {
     "nyc": "^14.1.1",
-    "vows": "^0.8.2"
+    "vows": "^0.8.3"
   },
   "engines": {
     "node": ">=4"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
```

Comment:
make it `devDependencies`. only those that work on the library will need to install it right? `typescript` dependency should also be on there